src/components/Editor/Preview/index.js

```diff
@@ -39,16 +39,29 @@ class Preview extends PureComponent {
     const self = this;
     self.onScroll = this.onScroll.bind(this);
     self.onMouseOver = debounce(this.onMouseOver, 40);
+    self._onMouseOver = e => this.onMouseOver(e);
+    self._onMouseUp = e => this.onMouseUp(e);
+    self._onMouseDown = e => this.onMouseDown(e);
```

Comment:
can we bind here: `self.onMouseDown = this.onMouseDown.bind(this)`

Reply:
Sure. Will update.